packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -135,7 +135,7 @@ def get_chart_from_oci(tmpdir, release, repository = None, version = None):
             output = subprocess.check_output(cmnd, stderr=subprocess.STDOUT, cwd=tmpdir, shell=True)
             logger.info(output)
 
-            return os.path.join(tmpdir, release)
+            return os.path.join(tmpdir, repository.rpartition('/')[-1])
```

Comment:
This is a breaking change. Unfortunately, the convention that this establishes differs from the helm convention. Please add an option to `KubectlProvider` (I think that's the one that creates this lambda) to allow users to specify the path. This option should be the final file path, so we don't to have to decode it (calling `rpartition()` with -1) in the lambda handler.

Reply:
Can you elaborate on how it is a breaking change? It breaks compatibility? With what? As i can see - how chart is placed in the filesystem during lambda runtime it's just an implementation detail which doesn't leak anywhere.

Reply:
if they have a deployment that loads that chart from `release`, then this change will break them. The chart will no longer be in `release`, so this needs to be behind a prop, like `chartDirectory`

Reply:
Ahhh I see, this is actually a safe change. Can you add comments above this `rpartition()` cal explaining that this is where `helm pull` saves the chart?